pydle/client.py

```diff
@@ -365,7 +345,17 @@ def _parse_message(self):
     async def handle_forever(self):
         """ Handle data forever. """
         while self.connected:
-            data = await self.connection.recv()
+            try:
+                data = await self.connection.recv(timeout=self.READ_TIMEOUT)
+            except asyncio.TimeoutError:
+                self.logger.debug('>> Receive timeout reached, sending ping to check connection state...')
```

Comment:
I think this should be emitted as a warning, as a read timeout is something that should warrant investigation
```suggestion
                self.logger.warning('>> Receive timeout reached, sending ping to check connection state...')
```

Reply:
Fair suggestion!